packages/destination-actions/src/destinations/fullstory/trackEventV2/generated-types.ts

```diff
@@ -0,0 +1,30 @@
+// Generated file. DO NOT MODIFY IT BY HAND.
+
+export interface Payload {
```

Comment:
Is this file intended to be included as part of this PR? It looks like this PR only adds `identifyUserV2`.

Reply:
must have accidentally grabbed that file while working on events.